parser/statement.go

```diff
@@ -218,12 +262,21 @@ func (self *_parser) parseFunctionBlock(node *ast.FunctionLiteral) {
 func (self *_parser) parseDebuggerStatement() ast.Statement {
 	idx := self.expect(token.DEBUGGER)
 
+	comments := self.findComments(true)
+
 	node := &ast.DebuggerStatement{
 		Debugger: idx,
 	}
 
+	self.commentMap.AddComments(node, comments, ast.TRAILING)
+
 	self.semicolon()
 
+	if !self.skippedLineBreak {
+		comments = self.findComments(false)
```

Comment:
I just try not to have too many function calls in one line. I'm not religious about it, though.